example/wde/main.go

```diff
@@ -2,7 +2,7 @@ package main
 
 import (
 	"fmt"
-	"github.com/skelterjohn/go.wde"
+	"github.com/skelterjohn/go.wde" // nolint: goimports
```

Comment:
I don't love the magic comment here, it's a perfectly normal import?

Reply:
One would think, yes - sadly, the linter has a standing issue to trip up with special characters in package names (such as `.` or `-`) and then complains about wrong import sorting. :/